schemas/2017-12-01-preview/Microsoft.DBforPostgreSQL.json

```diff
@@ -342,11 +470,15 @@
               "type": "string",
               "enum": [
                 "9.5",
-                "9.6"
+                "9.6",
+                "10",
+                "10.0",
```

Comment:
We'd like to hide these two: 10.0 and 10.2. We'd like to have only 10.

Reply:
Swagger is the main API spec for your RP - technically you're already declaring support for 10.2 here. If you need to fix this then I'd suggest submitting a PR to this repo:
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL

Reply:
Thanks! Ife has informed me of this. We're good here.